shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/ClangArgs.kt

```diff
@@ -69,6 +69,12 @@ class ClangArgs(private val configurables: Configurables) : Configurables by con
                 KonanTarget.IOS_ARM64 ->
                     listOf("-stdlib=libc++", "-arch", "arm64", "-isysroot", absoluteTargetSysRoot, "-miphoneos-version-min=9.0.0")
 
+                KonanTarget.TVOS_ARM64 ->
+                    listOf("-stdlib=libc++", "-arch", "arm64", "-isysroot", absoluteTargetSysRoot, "-mtvos-version-min=12.2")
```

Comment:
Better use older version.

Reply:
Don't we have `osVersionMin.tvos_x64` and such?